default/stringtables/en.txt

```diff
@@ -4355,7 +4355,7 @@ RESEARCH_TEXT
 '''The Research (or RP "Research Points") of all planets in the empire is combined to discover new technologies. Any Research that cannot be spent on a tech is lost each turn. Each tech has a minimum number of turns required to research it. For example, if a tech that cost 15 RPs, and had a minimum time of 3 turns, 5 RPs is the maximum that could be put towards its completion per turn.
 The tech at the top of the queue is given RPs first. Any remaining RP are then given to the next tech in the queue, and if there is still more remaining to the next-- and so on. You can re-prioritize the items in the queue by dragging the most important items closer to the top.
 
-Research does not require [[encyclopedia SUPPLY_TITLE]] line connections to be used.'''
+Research does not require [[encyclopedia SUPPLY_TITLE]] line connections to be used. However, if a tech which improves a ship part is researched (e.g. a weapon), the already existing ships must be in supply range, stationary, and not in combat, in order to receive the upgrade during the following turn.'''
```

Comment:
This would make more sense as a new article titled "Technologies" or similar. The article being modified here is about the research output of planets, not what effects techs have once completed.